### ISSUE TITLE
Contrast issues in duration bar on /Traces

### ISSUE THREAD

**Issue by tlmii** [dotnet/aspire#689]
The contrast between the standard background and the duration bars on the /Traces page doesn't meet the normal minimum contrast requirements for user interface elements in either mode:

Dark Mode:
![image](https://github.com/dotnet/aspire/assets/9613109/7dd8a31d-bb2a-4d77-8862-f7e2e371069e)
Light mode:
![image](https://github.com/dotnet/aspire/assets/9613109/128c2bcc-6a06-4f7b-baf2-42248e83e0e6)

Additionally, the contrast between the subtext and the duration highlight color doesn't meet the minimum contrast requirements. This part is additionally complicated because we don't know whether the subtext is over, partially over, or not over the duration bar, and that occurrence changes as the user resizes the page and the columns:
![image](https://github.com/dotnet/aspire/assets/9613109/6f368ca1-7053-4ffc-b05b-87802e9c9e03)

A few things we need to do here:

1.  It may not be possible to pick a color for the color bars that has sufficient contrast with the background AND sufficient contrast with the foreground. But we may be able to ignore the contrast issue with the background if we are conveying the same information in another way on the page. We should see what other ways we could convey that information (I believe it is basically a ratio of the current row's duration to the longest duration) on the page and talk to the a11y folks to see if that is sufficient.
2. Even if we do the above, we'd still need to make sure that the text contrast with the duration background is sufficient.
